broker/src/main/java/io/camunda/zeebe/broker/transport/streamapi/StreamApiHandler.java

```diff
@@ -49,24 +53,31 @@ public void add(final MemberId sender, final AddStreamRequest request) {
     properties.wrap(request.metadata(), 0, request.metadata().capacity());
 
     if (request.streamType().capacity() <= 0) {
-      throw new IllegalArgumentException(
+      final String errorMessage =
           "Expected a stream type of length > 0, but it has %d"
-              .formatted(request.streamType().capacity()));
+              .formatted(request.streamType().capacity());
+      LOG.warn("Failed to open stream. {}", errorMessage);
+      throw new IllegalArgumentException(errorMessage);
```

Comment:
:question: Do you know if Atomix will log this before returning it? I can't remember. Would we get double logging?

Reply:
I guess it will be warned on the client side (i.e gateway). But since this is an unexpected case, we could err on the safe side and log multiple times.